src/Location/dto/location.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class LocationDto {
+  id:string;
+  name: string;
+  city: string;
+  address:string;
+  locationLicense: number;
```

Comment:
shouldn't the locationLicense be optional? a bus stations doesn't have a license

Reply:
because of that, I did it not required and the mongo schema

Reply:
so change it to `locationLicense?: number`